### ISSUE TITLE
Call on `RtcDataChannel<D>::receive` panicks

### ISSUE THREAD

**Issue by wngr** [lerouxrgd/datachannel-rs#20]
_First: Thanks for this crate, it has been really helpful for me!_

Calling `receive` on a `RtcDataChannel` panicks:
```
thread 'main' panicked at 'Couldn't get buffer size: NotAvailable', /home/ow/.cargo/registry/src/github.com-1ecc6299db9ec823/datachannel-0.6.4/src/datachannel.rs:216:10
```

I _think_ this is because the `OnMessage` callback is always set in this crate. If that is set, the `receive` function must not be called according to https://libdatachannel.org/pages/reference.html.
I think either the `receive` function should not be exposed at all, or at least return an error and not panic (not sure how this crate can be used w/o setting the `OnMessage` cb..).

**Comment by lerouxrgd** [comment]
Indeed good point, I will remove `receive` from the public API as it can be confusing.

**Comment by lerouxrgd** [comment]
Fixed in `0.7.0`.

**Comment by wngr** [comment]
Thanks for the quick action.
